tests/sqlite3/test-sqlite3.cpp

```diff
@@ -105,6 +105,35 @@ TEST_CASE("SQLite blob", "[sqlite][blob]")
     }
 }
 
+TEST_CASE("SQLite blob on a rowset", "[sqlite][blob][rowset]")
```

Comment:
Why should this be SQLite-specific? I think this should work for all backends, are there any known problems with this?

Reply:
Yes, it should and I am working on that (more info on the comments below here or on the issue 922).

At the time I pushed it here, it was working only on SQLite3. My goal was to have the VS integration compiling before moving further.

Talking about tests, what do you think about centralising the tests that should work on every backend? I would happily do it next, if you think it is appropriate.